python3Pyside2QML/Dockerfile.debug

```diff
@@ -69,9 +112,16 @@ RUN apt-get -q -y update && \
     python3-pyside2.qtquick \
     python3-pyside2.qtnetwork \
     qml-module-qtquick-dialogs \
+    qtwayland5 \
     && apt-get clean && apt-get autoremove && \
     rm -rf /var/lib/apt/lists/*
 
+# replace the libs from the opengles image
+COPY --from=opengles /usr/lib/aarch64-linux-gnu /usr2/lib/aarch64-linux-gnu
+# rsync the files
+RUN rsync -a /usr2/lib/aarch64-linux-gnu /usr/lib/aarch64-linux-gnu && \
```

Comment:
`rsync` needs to be added on the  `apt-get -q -y install ...` , after line `115`